admission-server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=${BUILDPLATFORM} node:lts AS builder
```

Comment:
This option _should_ be set for builders, it's what allows the cross compilation to be fast.

Reply:
I did a little more reading.  I'll modify the final host image platform to make sure the runtime looks right.  While I do that, do you know:

1.  If the GO compiler needs to be run through the target platform?
2. If there are any node modules that pull in architecture dependent native binaries?